src/Mumble.proto

```diff
@@ -553,6 +553,13 @@ message UserStats {
 	// True if the user has a strong certificate.
 	optional bool strong_certificate = 18 [default = false];
 	optional bool opus = 19 [default = false];
+
+	// Rolling packet statistics time window as defined on the server.
+	optional uint32 rolling_time = 20;
+	// Rolling packet statistics for packets received from the client.
+	optional Stats rolling_from_client = 21;
+	// Rolling packet statistics for packets sent by the server.
+	optional Stats rolling_from_server = 22;
```

Comment:
I wonder whether creating a dedicated sub-message that encapsulates all rolling stats related fields might make things a bit clearer (as we wouold no longer have to use the `rolling` prefix for all fields :thinking: